Finmer.Game/Models/UserConfig.cs

```diff
@@ -46,6 +46,11 @@ public enum EAnimationLevel
         /// </summary>
         public static bool PreferScat { get; set; } = true;
 
+        /// <summary>
+        /// Indicates whether PreySense is enabled prior to vore scenes
+        /// </summary>
+        public static bool PreySense { get; set; } = true;
```

Comment:
I'd prefer having this set to `false` by default. That way, players can enable content warnings if they so choose, but the "default experience" will allow for some surprises here and there.